.dockerignore

```diff
@@ -0,0 +1,25 @@
+.github/
+.ipynb_checkpoints/
+.vscode/
+bin/
+docs/
+venv/
```

Comment:
Having a .dockerignore that is economical in terms of what is sent to the docker context is important because it will speed up your `docker build` commands.